### PR TITLE
[Bulk] Validation: Fixed missing alttext 1

### PR DESCRIPTION
**Pull request by reemsabaw1**
Fixes #665

### CODE REVIEW

File: aspnet/mvc/overview/older-versions-1/nerddinner/build-a-model-with-business-rule-validations.md [hunk 1]
```diff
@@ -43,41 +43,41 @@ We can use the LINQ language support within VB and C# to write expressive querie
...
 
-![](build-a-model-with-business-rule-validations/_static/image6.png)
+![Screenshot of the Dinner and R S V P tables. An arrow is highlighted and pointing from the Dinner properties tree and R S V P properties tree.](build-a-model-with-business-rule-validations/_static/image6.png)
 
 The above association will cause LINQ to SQL to add a strongly typed "Dinner" property to the RSVP class that developers can use to access the Dinner associated with a given RSVP. It will also cause the Dinner class to have a "RSVPs" collection property that enables developers to retrieve and update RSVP objects associated with a particular Dinner.
 
 Below you can see an example of intellisense within Visual Studio when we create a new RSVP object and add it to a Dinner's RSVPs collection. Notice how LINQ to SQL automatically added a "RSVPs" collection on the Dinner object:
 
-![](build-a-model-with-business-rule-validations/_static/image7.png)
+![Screenshot of the intellisense within Visual Studio. R S V Ps is highlighted.](build-a-model-with-business-rule-validations/_static/image7.png)
```
Comment (Roudabeh1348) [inline aspnet/mvc/overview/older-versions-1/nerddinner/build-a-model-with-business-rule-validations.md:76]: R S V Ps or R S V Ps collection?

Comment (reemsabaw1) [reply aspnet/mvc/overview/older-versions-1/nerddinner/build-a-model-with-business-rule-validations.md:76]: R S V Ps is what is shown in the image. :)

File: aspnet/mvc/overview/older-versions-1/nerddinner/enable-automated-unit-testing.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ Visual Studio 2008 Professional (and higher editions) includes a built-in test r
 
 Let's position our cursor within the DinnerTest class and type "Ctrl R, T" to run the two tests we just defined. When we do this a "Test Results" window will appear within Visual Studio and we'll see the results of our test run listed within it:
 
-![](enable-automated-unit-testing/_static/image5.png)
+![Sc](enable-automated-unit-testing/_static/image5.png)
```
[the remark this answers is not in the document]
Comment (reemsabaw1) [reply aspnet/mvc/overview/older-versions-1/nerddinner/enable-automated-unit-testing.md:90]: Good catch! Thank you.

File: aspnet/mvc/overview/older-versions-1/nerddinner/implement-efficient-data-paging.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ We can use the exact same Index() code we had before with our Querystring scenar
 
 And now when we run the application and type in */Dinners* we'll see the first 10 upcoming dinners:
 
-![](implement-efficient-data-paging/_static/image3.png)
+![Screenshot of the ](implement-efficient-data-paging/_static/image3.png)
```
[the remark this answers is not in the document]
Comment (reemsabaw1) [reply aspnet/mvc/overview/older-versions-1/nerddinner/implement-efficient-data-paging.md:88]: Thanks for catching this!

**Comment by reemsabaw1** [comment]
@Rick-Anderson 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**